ank/src/cli_commands.rs

```diff
@@ -223,42 +224,52 @@ impl CliCommands {
         let _ = self.task.await;
     }
 
+    async fn wait_for_complete_state(&mut self, object_field_mask: &Vec<String>, timeout_ms: u128) -> Result<Box<CompleteState>, CliError> {
```

Comment:
Maybe we can just call it `get_complete_state` and don't say anything about waiting. I also would leave it as you initially wrote it with the constant timeout instead of an input argument.